rocAL/rocAL/include/decoders/image/open_cv_decoder.h

```diff
@@ -45,7 +45,7 @@ class CVDecoder : public Decoder {
      \param color_comps pointer to the user's buffer to write the number of color components of the compressed image to 
     */
     Status decode_info(unsigned char* input_buffer, size_t input_size, int* width, int* height, int* color_comps) override;
-	
+    
```

Comment:
Please leave exact number of spaces, to avoid showing this as a separate file change